pkg/reconciler/v1alpha1/revision/revision.go

```diff
@@ -51,6 +51,7 @@ import (
 	listers "github.com/knative/serving/pkg/client/listers/serving/v1alpha1"
 	"github.com/knative/serving/pkg/reconciler"
 	"github.com/knative/serving/pkg/reconciler/v1alpha1/revision/config"
+	netcfg "github.com/knative/serving/pkg/reconciler/v1alpha1/route/config"
```

Comment:
ew, can you move to `pkg/network/config.go` instead?  Then it could be `network.Config{}`

Reply:
Done